lib/utils.ts

```diff
@@ -3,7 +3,8 @@ import unfetch from 'unfetch'
 
 export function checkMountModeEnabled() {
   // @ts-ignore
-  if (Cypress.spec.specType !== 'component') {
+  const specType = Cypress?.spec?.specType // this will be present for cypress@6.x, but not for cypress@7.x
+  if (specType != null && specType !== 'component') {
```

Comment:
@bahmutov, `Cypress.spec.specType` is `undefined` and it may suffice to use `Cypress.testingType`

cc @JohnnyFun

Reply:
Another angle is that Cypress may no longer provide the `specType` attribute since it now has dedicated commands for component testing, `open-ct` and `run-ct`. And looking at the output from [here](https://docs.cypress.io/api/cypress-api/spec#Syntax), also shown below, it is safe to use `Cypress.testingType`

```js
it('log spec info', () => {
  console.log(Cypress.spec)
  // {
  //   name: 'filter.spec.js',
  //   relative: 'cypress/integration/filter.spec.js',
  //   absolute: '/Users/janelane/Dev/web-app/cypress/integration/filter.spec.js',
  // }
})
```